specs/_features/maxeb_increase/capella.py

```diff
@@ -236,7 +236,9 @@ def floorlog2(x: int) -> uint64:
 MAX_BLS_TO_EXECUTION_CHANGES = 16
 MAX_WITHDRAWALS_PER_PAYLOAD = uint64(16)
 MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP = 16384
-
+DOMAIN_CONSOLIDATION = DomainType('0x0B000000')
+MAX_CONSOLIDATIONS = 4
```

Comment:
where does the 4 consolidations per slot come from?

Reply:
also what happens if more consolidations are requested than fit in the pending list